docs/content/common-subs/required-packages.md

```diff
@@ -4,6 +4,9 @@
         ``` title="jq - https://stedolan.github.io/jq/download/"
         brew install jq
         ```
+        ``` title="yq - https://github.com/mikefarah/yq#install"
+        brew install yq
+        ```
         ``` title="docker - https://docs.docker.com/engine/install/"
```

Comment:
Some day we should advertise podman as a supported alternative to docker.
But that is not a matter for this PR.